src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -2361,14 +2363,21 @@ import Color from 'color';
 
   InCHlib.prototype._draw_heatmap_header = function () {
     const self = this;
-    if (self.options.heatmap_header && self.header.length > 0) {
+    if (self.options.heatmap_header &&
+      self.header.length > 0 &&
+      self.pixels_for_dimension >= self.min_size_draw_values) {
```

Comment:
```suggestion
      self.pixels_for_dimension >= self.min_size_draw_values
    ) {
```

Reply:
duly noted, will add to new branch as discussed